invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -96,40 +96,37 @@ <h2>{{ add_description.type.title_l10n }} <span
 
 
 {% macro _show_funding_item(item, index) %}
+  <dt class="ui tiny header">{{ item.funder.name if item.funder }}</dt>
   {%- if item.award -%}
-    {%- if item.award.title_l10n -%}
-      <dt class="ui tiny header">
-        <span class="mr-5">
-          {% if item.award.acronym %}
-            {{ item.award.acronym }} –
-          {% endif %}
-
-          {{ item.award.title_l10n }}
-        </span>
+  <dt class="ui tiny header">
```

Comment:
Maybe <dt class="text-muted"> for the title? I'm thinking there should be some style differentiation between the title and funder.